crogine/CMakeLists.txt

```diff
@@ -156,6 +154,7 @@ if(NOT TARGET crogine)
     add_library(${PROJECT_NAME} SHARED ${PROJECT_SRC})
   endif()
 
+  target_compile_definitions(${PROJECT_NAME} PUBLIC $<$<CONFIG:Debug>:CRO_DEBUG_>)
```

Comment:
This is public so that targets depending on crogine automatically get the define - Currently doesn't make a huge amount of difference as you're still sometimes building golf as a standalone project, so golf still needs to add the definition itself, but down the line that could be improved